docs/appendix/zowe-security-glossary.md

```diff
@@ -1,4 +1,4 @@
-# Zowe security concepts glossary
+# Glossary of Zowe Security terminology
```

Comment:
It may be more consistent if security is lowercase.

Reply:
I mistakenly based this branch on `master` instead of `docs-staging` so you're seeing a lot of "changes" that I didn't make, including this one. I'm not editing this content as this is another writer's work and the eventual merge to the `master` branch (when we publish to the live site) may well end up fixing this.